.github/workflows/unit-tests.yaml

```diff
@@ -82,12 +82,17 @@ jobs:
     strategy:
       matrix:
         gotags: [ 'GOTAGS=""', 'GOTAGS=release' ]
+        pg: [ '13', '15' ]
     runs-on: ubuntu-latest
     outputs:
       new-jiras: ${{ steps.junit2jira.outputs.new-jiras }}
     container:
-      image: quay.io/stackrox-io/apollo-ci:stackrox-test-0.4.5
+      image: quay.io/stackrox-io/apollo-ci:stackrox-test-0.4.6
```

Comment:
@erthalion Could you upgrade other places as well. We try to keep this version in sync in all jobs.